Formula/openjdk.rb

```diff
@@ -73,28 +82,60 @@ def install
                 .max
     raise "cannot find build number in .hg_archival.txt" if build.nil?
 
+    args = %W[
+      --disable-warnings-as-errors
+      --with-boot-jdk-jvmargs=#{java_options}
+      --with-boot-jdk=#{boot_jdk}
+      --with-debug-level=release
+      --with-jvm-variants=server
+      --with-native-debug-symbols=none
+      --with-toolchain-path=/usr/bin
```

Comment:
Seems wrong to hardcode this

Reply:
There is no change from current behavior.